public/docs/ts/latest/tutorial/index.jade

```diff
@@ -2,80 +2,82 @@ block includes
   include ../_util-fns
 
 :marked
-  Our grand plan for this tutorial is to build an app to help a staffing agency manage its stable of heroes.
-  Even heroes need to find work.
+  The grand plan for this tutorial is to build an app that helps a staffing agency manage its stable of heroes.
 
-  Of course we'll only make a little progress in this tutorial. What we do build will
-  have many of the features we expect to find in a full-blown, data-driven application: acquiring and displaying
+  The Tour of Heroes app covers the core fundamentals of Angular. You'll build a basic app that
+  has many of the features you'd expect to find in a full-blown, data-driven app: acquiring and displaying
   a list of heroes, editing a selected hero's detail, and navigating among different
   views of heroic data.
 
-  The Tour of Heroes covers the core fundamentals of Angular.
-  We’ll use built-in directives to show/hide elements and display lists of hero data.
-  We’ll create a component to display hero details and another to show an array of heroes.
-  We'll use one-way data binding for read-only data. We'll add editable fields to update a model
-  with two-way data binding. We'll bind component methods to user events like key strokes and clicks.
-  We’ll learn to select a hero from a master list and edit that hero in the details view. We'll
-  format data with pipes. We'll create a shared service to assemble our heroes. And we'll use routing to navigate among different views and their components.
+  You'll use built-in directives to show and hide elements and display lists of hero data.
+  You'll create components to display hero details and show an array of heroes.
+  You'll use one-way data binding for read-only data. You'll add editable fields to update a model
+  with two-way data binding. You'll bind component methods to user events, like keystrokes and clicks.
+  You'll enable users to select a hero from a master list and edit that hero in the details view. You'll
+  format data with pipes. You'll create a shared service to assemble the heroes.
+  And you'll use routing to navigate among different views and their components.
+  <!-- CF: Should this be a bullet list? -->
 
-  We’ll learn enough core Angular to get started and gain confidence that
-  Angular can do whatever we need it to do.
-  We'll be covering a lot of ground at an introductory level but we’ll find plenty of links
-  to chapters with greater depth.
+  You'll learn enough core Angular to get started and gain confidence that
+  Angular can do whatever you need it to do.
+  You'll cover a lot of ground at an introductory level, and you'll find many links
+  to pages with greater depth.
+
+  When you're done with this tutorial, the app will look like this <live-example name="toh-6"></live-example>.
```

Comment:
Let's just call it plunker. "People know what it is (we've told them). You can run the live example of the app in plunker and download the code from there."